NeuroID/src/main/AndroidManifest.xml

```diff
@@ -5,4 +5,5 @@
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
+    <uses-permission android:name="android.permission.READ_PHONE_STATE" />
```

Comment:
I think this should be only in the app level (sample app). The docs will explain to the user that if they want to pull these signals to add the permission to their app and request permissions from the user. Looks like the sample app already has this in there.